tests/integration/Tribe/Events/Integrations/QR/RedirectionsTest.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+namespace Tribe\Events\Integrations\QR;
+
+use Codeception\TestCase\WPTestCase;
+use TEC\Events\QR\Redirections;
+use TEC\Events\QR\Routes;
+use TEC\Events\QR\Settings;
+use Tribe__Events__Main as TEC;
+
+/**
+ * Tests QR Redirections functionality
+ *
+ * @group   core
+ * @group   qr
+ *
+ * @package TribeEvents
+ */
+class RedirectionsTest extends WPTestCase {
```

Comment:
Since we are testing controllers shouldnt we extend the Controller Test case ? And make use of the advantages that come with like `$this->make_controller()->register()` ?

Using the controller test case has also other advantages like using test_services which is a clone of our production controller. Making registrations to the test controller during tests which makes for cleaner testing :D